botfront-project/templates/ingress.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
   name: {{ .Release.Name }}-rasa-ingress
   annotations:
     {{- if .Values.ingress.nginx.enabled}}
+    #uncomment the following line if you are working with multiple rasa using a websocket based channel
+    # web socket connection need to have a sticky session https://socket.io/docs/using-multiple-nodes/
+    #nginx.ingress.kubernetes.io/affinity: "cookie"
```

Comment:
I think this should be a  `ingress.nginx.enableSessionAffinity`parameter enabled by default. The line should be surrounded by conditionals so it is only added if enabled. 
The documentation should mention that this should be enabled if a channel depending on websockets or long polling (such as Rasa Webchat) is used.

Reply:
Move it to the table for rasa values